### DISCUSSION TITLE
How to modify location address?

### DISCUSSION THREAD

**Discussion by dzoey** [teslamate-org/teslamate#1511] [Q&A]
When looking at the trips my Model 3 takes it gives the address of my house incorrectly as one of the houses down the street.  The address is at least 50 yards away.  Coincidentally, I think it's the other person on my street with a Tesla, but I don't know if that's important or not.   When Teslamate shows me my car on the map, the location is correct, so it's not GPS that's off.  The car is shown at the correct house, it's just the house number that's being reported is wrong.

Where does it get the location from?  Is it the car that's reporting the address or is Teslamate looking it up incorrectly?  Is there some way of correcting it so the right house number is used?

Update:  One work around is click on the address in Locations and provide a name for the address.   It doesn't actually fix the underlying incorrect address problem but now instead of showing the incorrect house number, it shows "home".  This is good enough, I think.

**Comment by eliluong** [comment]
It takes the GPS coord of the vehicle, then runs it against an API to generate an address. Sometimes correct, sometimes incorrect. So you can create a geofence and put either 'home' like you did, or you can manually type in the address if you always park in the same spot.

**Reply by bahree** [reply]
I also notice this happens when the car has just woken up and I drive away - I presume it is the delta from when the API's pick it up and start recording that. I normally wake up the car a few mins before heading out and the latency has been reduced and it is not a problem. Maybe something to try if you haven't already.
